src/Orc.FileAssociation.Tests/FileAssociationsServiceFacts.cs

```diff
@@ -21,6 +24,20 @@ public async Task AssociateFilesWithApplicationAsyncTestAsync(ApplicationInfo ap
             Assert.AreEqual(expectedName, fileAssociationServiceMock.Name);
         }
 
+        [Test]
+        [Explicit]
+        public async Task OpenPropertiesWindowForExtensionAsyncTestAsync()
+        {
+            var path = $"{Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)}\\WildGums\\Temp\\";
+            var filePath = Path.Combine(path, "Click on 'Change' to select default csv handler.csv");
+            GetServices();
+            var fileAssociationServiceMock = new FileAssociationServiceMock(FileService, DirectoryService);
+            await fileAssociationServiceMock.OpenPropertiesWindowForExtensionAsync("csv", path);
```

Comment:
Should we use path as TestCase parameter? Easy to extend if some more cases need in the future